apps/src/javalab/Playground.js

```diff
@@ -103,4 +132,18 @@ export default class Playground {
       return;
     }
   }
+
+  getUrl(filename) {
```

Comment:
Do we assume Javabuilder has already done the work of confirming that the file exists?

Reply:
So eventually, yes, Javabuilder would have confirmed that this file exists in the student's project. We could also do some client-side validation though - I'll add an onerror callback we can use